modules/openapi-generator/src/test/resources/3_0/python/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -2711,3 +2711,26 @@ components:
       properties:
         field:
           type: string
+    CircularAllOfRef:
+      allOf:
+        - $ref: '#/components/schemas/AllOfSuperModel'
+      properties:
+        secondCircularAllOfRef:
+          items:
+            $ref: '#/components/schemas/SecondCircularAllOfRef'
+          type: array
+      type: object
+    SecondCircularAllOfRef:
+      allOf:
+        - $ref: '#/components/schemas/AllOfSuperModel'
+      properties:
+        circularAllOfRef:
+          items:
+            $ref: '#/components/schemas/CircularAllOfRef'
+          type: array
+      type: object
+    AllOfSuperModel:
+      properties:
+        _name:
+          type: string
+      type: object
```

Comment:
for the tests, can you please put properties under alLOf instead as the python client tests do not have any normalizer rule enabled?

Reply:
can you also add a test in https://github.com/OpenAPITools/openapi-generator/blob/99ca559fd25d8ba20a472a66435e083583bdee70/samples/openapi3/client/petstore/python/tests/test_model.py by actually importing the model and use it so as to ensure there will be no exception when using the model?

Reply:
Thanks @wing328 , all comments addressed.